graph/src/data/subgraph/mod.rs

```diff
@@ -898,27 +900,51 @@ impl UnresolvedSubgraphManifest {
             repository,
             schema,
             data_sources,
+            templates,
         } = self;
 
+        match semver::Version::parse(&spec_version) {
+            Ok(ref ver) if *ver <= semver::Version::new(0, 0, 2) => {}
+            _ => {
+                return Box::new(future::err(format_err!(
+                    "This Graph Node only supports manifest spec versions <= 0.0.2,
+                but subgraph `{}` uses `{}`",
```

Comment:
Is the message formatting / indentation correct here? Looks like it'll print an error message with the second line way indented.

Reply:
This should print as a single line with a single space in the line break, but I've aligned it nicely.